unit_testing/Util/serialStatus_tests.cpp

```diff
@@ -0,0 +1,109 @@
+/*!
+** @file   ll_tests.cpp
+** @author Luke W
+** @date   26/07/2024
+*/
+
+#include <vector>
+
+#include "fake_USBprint.h"
+#include "serialStatus_tests.h"
+#include "pcbversion.h"
+
+using namespace std;
+
+/***************************************************************************************************
+** TESTS
+***************************************************************************************************/
+
+void goldenPathTest(SerialStatusTest& sst, const char* pass_string, int firstPrintTick) {
```

Comment:
I think all the general test functions should have clear comments stating their purpose :=)